bigtable/admin.go

```diff
@@ -310,8 +310,9 @@ type TableInfo struct {
 
 // FamilyInfo represents information about a column family.
 type FamilyInfo struct {
-	Name     string
-	GCPolicy string
+	Name          string
+	GCPolicy      string
+	GCPolicyProto *btapb.GcRule
```

Comment:
Have you considered just adding the fields you need directly on this object, rather than exposing the whole proto?

In general we try to avoid exposing proto-types directly via the manual clients. Though, I see a few cases already where they are in this library, so I'll just have this as a question/suggestion rather than being a purist about it.

Reply:
We're interested in recursively iterate through this proto to generate a valid JSON field for Terraform. The one we're interested in is  the Rule object in *btapb.GcRule, but the type of that object is not exported, so I opt in to pass the entire proto here.

Reply:
I agree with @tritone. We typically try to provide a veneer over proto message types.

Reply:
I see, so how about adding a new struct in the bigtable client and a parser function to convert the proto to the new struct, and pass that struct back from the admin?